readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -2,28 +2,11 @@
  * Sphinx and Mkdocs search overrides
  */
 
-var rtddata = require('./rtd-data');
-var xss = require('xss/lib/index');
-var MAX_RESULT_PER_SECTION = 3;
-var MAX_SUBSTRING_LIMIT = 100;
-
-/**
- * Create and return DOM nodes with given attributes.
- *
- * @param {String} nodeName name of the node
- * @param {Object} attributes obj of attributes to be assigned to the node
- * @return {Object} DOM node
- */
-const createDomNode = (nodeName, attributes) => {
-    let node = document.createElement(nodeName);
-    if (attributes) {
-        for (let attr of Object.keys(attributes)) {
-            node.setAttribute(attr, attributes[attr]);
-        }
-    }
-    return node;
-};
-
+const rtddata = require('./rtd-data');
+const xss = require('xss/lib/index');
+const { createDomNode } = require("./utils");
```

Comment:
Do we need to specify that we're using this function from `utils`, or is the compiler smart enough to know what is happening if we add additional utils functions?

Reply:
The compile step will resolve the functions in `utils.js` for use here, yeah. This syntax will only import `createDomNode` from `utils.js`. If more functions are added to `utils.js`, they won't automatically be imported here, but could be with `const { createDomNode, someOtherUtil } = ..`